std/range/package.d

```diff
@@ -3945,6 +3945,10 @@ private alias lengthType(R) = typeof(R.init.length.init);
     Throws:
         An `Exception` if all of the _ranges are not the same length and
         `sp` is set to `StoppingPolicy.requireSameLength`.
+
+    Limitations: Compiler always has to assume use of garbage collection
```

Comment:
I think we should word this in the opposite way for clarity. Ex:

```
Limitations: the compiler cannot guarantee @nogc or nothrow for this function because StoppingPolicy is not known at compile time.
```

Thoughts?

Reply:
Word order side, I like the phrasing "cannot infer" more than "cannot guarantee".

Reply:
Lol, you do all the hard work for me making the phrases. Perhaps that means I'm doing a bit too small PRs.